src/index.html

```diff
@@ -7,12 +7,67 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="/src/styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="/src/images/imac.jpeg"
+        alt="imac"
+        class="card__img"
+      />
+
+      <h2 class="card__text">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <h3 class="card__code">Product code: 195434</h3>
+
+      <div class="stars">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__reviews">Reviews: 5</div>
+      </div>
+
+      <div class="card__price">
+        <span class="card__price-text">Price:</span>
+        <span class="card__price-value">$2,199</span>
+      </div>
+
+      <div
+        class="card__link"
+        data-qa="hover"
+      >
+        <a
+          href="#"
```

Comment:
The href attribute of the anchor tag is set to '#'. This is generally used as a placeholder and should be replaced with a valid URL or removed if not needed.